### ISSUE TITLE
release with gin v1.4.0

### ISSUE THREAD

**Issue by jerome-laforge** [gin-contrib/pprof#14]
Hello,
could you, plz, release a new version that requires gin gonic v1.4.0?

because I run again in this problem:
```
 cannot load github.com/ugorji/go/codec: ambiguous import: found github.com/ugorji/go/codec in multiple modules:
        github.com/ugorji/go v1.1.4 (/somewhere/pkg/mod/github.com/ugorji/go@v1.1.4/codec)
        github.com/ugorji/go/codec v0.0.0-20181209151446-772ced7fd4c2 (/somewhere/pkg/mod/github.com/ugorji/go/codec@v0.0.0-20181209151446-772ced7fd4c2)
```
https://groups.google.com/forum/#!topic/golang-nuts/0mJh8SkaomA

**Comment by cupen** [comment]
+1

**Comment by ahdai0718** [comment]
Fix it in temporarily way above.

https://github.com/gin-gonic/gin/issues/1673
https://github.com/gin-gonic/gin/issues/1897

And it works for me 
```
replace github.com/ugorji/go v1.1.4 => github.com/ugorji/go v0.0.0-20190204201341-e444a5086c43

**Comment by sdgmf** [comment]
+1

**Comment by jerome-laforge** [comment]
fix with [v1.2.1](https://github.com/gin-contrib/pprof/blob/v1.2.1/go.mod)
